files/grest/rpc/transactions/tx_info.sql

```diff
@@ -63,25 +73,93 @@ BEGIN
           UNNEST(_tx_hashes) AS hashes)) T1
   LEFT JOIN LATERAL (
     SELECT
-      JSON_AGG(JSON_BUILD_OBJECT('index', tx_out.index, 'address', tx_out.address, 'value', tx_out.value)) as outputs
+      JSON_AGG(JSON_BUILD_OBJECT(
+        'payment_addr', json_build_object(
+          'bech32', tx_out.address,
+          'cred', ENCODE(tx_out.payment_cred, 'hex')
+          ),
+        'stake_addr', CASE WHEN SA.view IS NULL THEN NULL ELSE JSON_BUILD_OBJECT(
+          'bech32', SA.view
+          ) END,
+        'tx_hash', T1.tx_hash,
+        'tx_index', tx_out.index, 
+        'address', tx_out.address, 
```

Comment:
This field is duplicated in the JSON above:

'payment_addr', json_build_object(
          'bech32', tx_out.address,
          'cred', ENCODE(tx_out.payment_cred, 'hex')
          )